### PR TITLE
Update deploy/vercel.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

- Update deploy/vercel.mdx

This section is outdated, the current Astro version is 4.7.1 and I think we should update it

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by liruifengv** [comment]
netlify has the same issue

File: src/content/docs/en/guides/deploy/vercel.mdx [hunk 1]
```diff
@@ -6,6 +6,7 @@ i18nReady: true
 ---
 import ReadMore from '~/components/ReadMore.astro';
 import { Steps } from '@astrojs/starlight/components';
+import Version from '~/components/Version.astro';
```
Comment (sarah11918) [inline src/content/docs/en/guides/deploy/vercel.mdx:9]:
```suggestion
```

**Review by sarah11918 — APPROVED** [review]
Thanks @liruifengv ! As discussed on Discord, this doesn't even belong here any more! It was from before we had good upgrade guides.

To save us "upgrading" this section every time, we should just delete it and let the new Updating Astro page do this work!  :muscle: 

So let's just delete it entirely, and go ahead and merge this when you're ready!

**Review by sarah11918 — APPROVED** [review]
Thank you @liruifengv ! As discussed, we now have an Upgrade Astro page for all this information where it can be more easily kept up to date in one place. So lets just deleted it, then go ahead and merge when you're ready!

**Review by sarah11918 — APPROVED** [review]
Thank you @liruifengv ! As discussed, we now have an Upgrade Astro page for all this information where it can be more easily kept up to date in one place. So lets just deleted it, then go ahead and merge when you're ready!

**Review by dreyfus92 — APPROVED** [review]
Thanks everyone! LGTM ✅

**Pull request merged**